src/worker.py

```
@@ -52,6 +52,10 @@
 from pyodide.ffi import to_js
 from js import WebSocketPair, WebSocketRequestResponsePair
 import uuid
+try:
+    from chat_do import ChatDO
+except ImportError:
+    from src.chat_do import ChatDO
```

Flake8 (7.3.0)

[error] 'src.chat_do.ChatDO' imported but unused

(F401)